notebooks/302-pytorch-quantization-aware-training/302-pytorch-quantization-aware-training.ipynb

```diff
@@ -127,7 +127,8 @@
     "set_log_level(logging.ERROR)  # Disables all NNCF info and warning messages.\n",
```

Comment:
<p>Please add link on documenation (like it was before for Model Optimizer Developer Guide)</p><p>https://docs.openvino.ai/latest/openvino_docs_MO_DG_Python_API.html</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/918/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='32'/>

Reply:
Added.